src/main.ts

```diff
@@ -44,6 +46,20 @@ async function bootstrap() {
   const document = SwaggerModule.createDocument(app, config);
   SwaggerModule.setup('api', app, document);
 
+  // Register fastify-multipart plugin
+  app.register(fastifyMultipart);
+
+  // Access the Fastify instance to define the health check endpoint
+  const fastifyInstance = app.getHttpAdapter().getInstance();
+
+  fastifyInstance.get('/ping', async (request: FastifyRequest, reply: FastifyReply) => {
```

Comment:
No need to write complicated code for normal ping API write this in controller rather in main.ts file